.github/workflows/maven.yml

```diff
@@ -21,34 +21,45 @@ on: [pull_request]
 
 jobs:
   linux:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
         java: [ 8, 11, 14 ]
     steps:
     - uses: actions/checkout@v1
-    - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
+
+    - name: Install hava ${{ matrix.java }}
+      uses: actions/setup-java@v2
       with:
+        distribution: temurin
         java-version: ${{ matrix.java }}
+
     - name: Build with Maven
       run: ./mvnw clean verify -B -ntp -Dinvoker.streamLogs=true
```

Comment:
I'm not really sure why this don't work.
But after googling the error I found this:
https://stackoverflow.com/questions/50104172/could-not-find-or-load-main-class-org-apache-maven-wrapper-mavenwrappermain